app/views/records/edit_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= f.input :rights, as: :select_with_modal_help,
```

Comment:
@njaffer The modal help allows for you to display more information about the licenses: 
![screen shot 2016-02-22 at 11 05 25 am](https://cloud.githubusercontent.com/assets/1599081/13223927/4f47cdb8-d954-11e5-94e1-5d84622a3b31.png)

I'm not sure why that is not working, but removing it seems like a bad idea.